onnx/external_data_helper.py

```diff
@@ -36,13 +36,11 @@ def __init__(self, tensor):  # type: (TensorProto) -> None
 def load_external_data_for_tensor(tensor, base_dir):  # type: (TensorProto, Text) -> None
     """
     Load data from an external file for tensor.
-
+    Even if the tensor has raw_data, it will load from external data and override existing raw_data anyway.
```

Comment:
May be change to something like: Ideally TensorProto should not hold any raw data but if it does it will be ignored.